stages/apps/lead/variables.tf

```diff
@@ -75,6 +75,10 @@ variable "enable_harbor" {
   default = true
 }
 
+variable "enable_artifactory_jcr" {
+  default = true
```

Comment:
I don't think this should be `true`.  lead-terraform's primary use case is still for our lab purposes, so we don't want to accidentally leave this on for a lab.  I'd prefer this be `false` and we can explicitly override this in lead-environments.

Reply:
still looking to get this resolved